src/app/space/create/deployments/components/deployment-card/deployment-card.component.html

```diff
@@ -26,7 +26,11 @@
       </ul>
     </div>
     <div [collapse]="collapsed">
-      <pfng-chart-sparkline [config]="config" [chartData]="data"></pfng-chart-sparkline>
+      <p>CPU Usage</p>
+      <pfng-chart-sparkline [config]="cpuConfig" [chartData]="cpuData"></pfng-chart-sparkline>
+      <br>
+      <p>Memory Usage</p>
+      <pfng-chart-sparkline [config]="memConfig" [chartData]="memData"></pfng-chart-sparkline>
```

Comment:
Can you include a screenshot so we can see what these labels look like?

Reply:
Sure thing. You can see an example with the attached file
![screen shot 2017-12-06 at 11 31 23 am](https://user-images.githubusercontent.com/17230733/33672945-868e1cf0-da79-11e7-8fc4-64c87908f141.png)